src/Issuer/IssuerBuilder.php

```diff
@@ -10,11 +10,11 @@
 
 final class IssuerBuilder implements IssuerBuilderInterface
 {
-    /** @var MetadataProviderBuilder|null */
-    private $metadataProviderBuilder;
+    /** @psalm-suppress PropertyNotSetInConstructor */
+    private ?MetadataProviderBuilder $metadataProviderBuilder;
 
-    /** @var JwksProviderBuilder|null */
-    private $jwksProviderBuilder;
+    /** @psalm-suppress PropertyNotSetInConstructor */
+    private ?JwksProviderBuilder $jwksProviderBuilder;
```

Comment:
If you set the nullable type you should set it to `null` without suppressing the error.